doc/userdoc/installation/cmake_options.rst

```diff
@@ -28,70 +28,116 @@ Options for configuring NEST
+|                                             | ';'. [default OFF, when ON, defaults to '-O3']                 |
++---------------------------------------------+----------------------------------------------------------------+
+| -Dwith-warning=[OFF|ON|<list;of;flags>]     | Enable user defined warnings. Separate  multiple flags by ';'. |
+|                                             | [default ON, when ON, defaults to '-Wall']                     |
++---------------------------------------------+----------------------------------------------------------------+
+| -Dwith-debug=[OFF|ON|<list;of;flags>]       | Enable user defined debug flags. Separate  multiple flags by   |
+|                                             | ';'. [default OFF, when ON, defaults to '-g']                  |
++---------------------------------------------+----------------------------------------------------------------+
+|-Dwith-intel-compiler-flags=[<list;of;flags>]| User defined flags for the Intel compiler. Separate multiple   |
+|                                             | flags by ';'. [defaults to '-fp-model strict']                 |
++---------------------------------------------+----------------------------------------------------------------+
+| -Dwith-libraries=[<list;of;libraries>]      | Link additional libraries. Give full path. Separate multiple   |
+|                                             | libraries by ';'. [default OFF]                                |
++---------------------------------------------+----------------------------------------------------------------+
+| -Dwith-includes=[<list;of;includes>]        | Add additional include paths. Give full path without '-I'.     |
+|                                             | Separate multiple include paths by ';'. [default OFF]          |
++---------------------------------------------+----------------------------------------------------------------+
+| -Dwith-defines=[<list;of;defines>]          | Additional defines, e.g. '-DXYZ=1'. Separate multiple defines  |
+|                                             | by ';'. [default OFF]                                          |
++---------------------------------------------+----------------------------------------------------------------+
+| -Dwith-version-suffix=[string]              | Set a user defined version suffix. [default '']                |
++---------------------------------------------+----------------------------------------------------------------+
+
+**Change NEST behavior:**
+
++--------------------------+----------------------------------------------------+
+| -Dtics_per_ms=[number]   |  Specify elementary unit of time. [default 1000.0] |
++--------------------------+----------------------------------------------------+
+| -Dtics_per_step=[number] |  Specify resolution. [default 100]                 |
++--------------------------+----------------------------------------------------+
+
+**Add user modules:**
```

Comment:
Combine this with the timers and tics above and the target split below into "NEST properties"